.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
 
 env:
-  NODE_VERSION: 18.17
+  NODE_VERSION: 18
```

Comment:
As noted in https://github.com/ijlee2/ember-container-query/pull/211#discussion_r1359792838, I had to fix the Node version to `18.17` (i.e. disallow `18.18`). Now, version `18.19` runs in CI.